src/Compose/Php/ExtensionResolver.php

```diff
@@ -159,6 +159,20 @@ public static function getConfig(): array
             'bcmath' => [
                 '>=7.0' => [self::EXTENSION_TYPE => self::EXTENSION_TYPE_CORE],
             ],
+            'blackfire' => [
+                '>=7.2' => [
+                    self::EXTENSION_TYPE => self::EXTENSION_TYPE_INSTALLATION_SCRIPT,
+                    self::EXTENSION_INSTALLATION_SCRIPT => <<< BASH
+curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/linux/amd64/$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;")
+mkdir -p /tmp/blackfire
+tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire
+mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get ('extension_dir');")/blackfire.so
+( echo extension=blackfire.so
+echo blackfire.agent_socket=tcp://blackfire:8707 ) > $(php -i | grep "additional .ini" | awk '{print $9}')/blackfire.ini
+rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz
+BASH
+                ]
+            ],
```

Comment:
As per Blackfire docs for Docker integration: https://blackfire.io/docs/integrations/docker/php-docker